internal/services/mssql/mssql_virtual_machine_resource.go

```diff
@@ -724,6 +769,70 @@ func flattenSqlVirtualMachineAutoPatching(autoPatching *sqlvirtualmachine.AutoPa
 		},
 	}
 }
+	
+func expandSqlVirtualMachineAssessmentSettings(input []interface{}) *sqlvirtualmachine.AssessmentSettings {
+	if len(input) == 0 {
+		return nil
```

Comment:
Return with `Enabled` set to `false` so that it can be disabled?
```suggestion
		return nil
```
```suggestion
		return &sqlvirtualmachine.AssessmentSettings{
            Enable:                        utils.Bool(false),
        }
```